subsys/dfu/dfu_target/Kconfig

```diff
@@ -121,10 +121,54 @@ config DFU_TARGET_SUIT
 	select SUIT_UTILS
 	select ZCBOR
 	select ZCBOR_CANONICAL
+	select FLASH
+	select STREAM_FLASH
+	select STREAM_FLASH_ERASE
 	select DFU_TARGET_STREAM
+	# Minimal SUIT configuration to support SUIT processing
+	imply SUIT_DIGEST
+	imply SUIT_CRYPTO
+	imply SUIT_PLAT_CHECK_COMPONENT_COMPATIBILITY
+	imply SUIT_PLAT_CHECK_CLASSES
+	imply SUIT_CHECK_IMAGE_MATCH
+	imply SUIT_STREAM
+	imply SUIT_STREAM_SOURCE_CACHE
+	imply SUIT_STREAM_SOURCE_MEMPTR
+	imply SUIT_STREAM_SINK_DIGEST
+	imply SUIT_STREAM_SINK_CACHE
+	imply SUIT_STREAM_SINK_FLASH
+	imply SUIT_MEMPTR_STORAGE
+	imply SUIT_PLATFORM
+	imply SUIT_PROCESSOR
+	imply SUIT_SINK_SELECTOR
 	help
 	  Enable support for updates using DFU target based on SUIT
 
+if DFU_TARGET_SUIT
+
+config DFU_TARGET_SUIT_CACHE_PROCESSING
+	bool "SUIT cache processing"
+	imply SUIT_CACHE
+	imply SUIT_CACHE_RW
```

Comment:
I've seen that you use `DFU_TARGET_SUIT_CACHE_PROCESSING` to check if `SUIT_CACHE_RW` API is available. Please add `depends on` entries.

Reply:
I will discuss it with @SylwesterKonczyk, probably I need to rework it.

Reply:
Reworked.